.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,26 @@
+name: after-push checks
```

Comment:
stranger thing happens in the PR)))
I believe we should create another one for adding ci/cd